src/matrices.jl

```diff
@@ -0,0 +1,66 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENCE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    UnitMatrix{T}(d)
+    UnitColumn{T}(d)
+    F{T}(d)
+    G{T}(N)
+    H{T}(d)
```

Comment:
Let's rename these to `FMatrix`, `GMatrix` and `HMatrix`. And then we create an alias `F, G, H` for the type and export it. See the I example in the standard library LinearAlgebra.